pkgs/tools/typesetting/tex/texlive/lmtx.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, cmake, texlive, ... }:
+stdenv.mkDerivation {
+  pname = "texlive-luametatex.bin";
+  version = "2.10.07";
+
+  src = fetchFromGitHub {
+    owner = "contextgarden";
+    repo = "luametatex";
+    # no tags
```

Comment:
could you please document how you picked this tag? does it match tlpdb? just so we know how to deal with this in the future

Reply:
This should also be added to UPGRADING.md. What I did is run `./luametatex --version` on the binary from tlnet and search the commit with the matching version in the repository.

Unfortunately this mostly negates your efforts of reducing the number of hardcoded version numbers :-(